test/fixtures/dialects/mysql/back_quoted_identifier.sql

```diff
@@ -0,0 +1,9 @@
+SELECT * FROM `foo`.`bar`;
```

Comment:
These are parser tests, to test if your change behaves as expected, you need to test the rules. See here: https://github.com/sqlfluff/sqlfluff/tree/main/test/fixtures/rules/std_rule_cases